packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx

```diff
@@ -33,24 +33,21 @@ const VideoItem: FC<VideoItemProps> = ({
     onClose();
   };
 
-  const timeoutIdRef = useRef<ReturnType<typeof setTimeout>>();
-  const tooltipRef = useCallback(
-    (target: Element | null) => {
-      clearTimeout(timeoutIdRef.current);
-      ReactTooltip.hide(target ?? undefined);
-      if (target && !hasSeenTutorialsTooTip) {
-        timeoutIdRef.current = setTimeout(() => {
-          ReactTooltip.show(target);
-        }, 5_000);
-      }
-    },
-    [hasSeenTutorialsTooTip]
-  );
+  const ref = useRef<HTMLDivElement | null>(null);
+  const setRef: RefCallback<HTMLDivElement> = useCallback((node) => {
+    if (ref.current) {
+      return;
+    }
+    if (node && !hasSeenTutorialsTooTip) {
+      setTimeout(() => ReactTooltip.show(node), 5000);
+    }
+    ref.current = node;
+  }, []);
```

Comment:
OPT:

> Check warning on line 45 in packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx
GitHub Actions
/ linting
React Hook useCallback has a missing dependency: 'hasSeenTutorialsTooTip'. Either include it or remove the dependency array